x-pack/plugins/security_solution/server/lib/sourcerer/routes/index.ts

```diff
@@ -26,85 +34,214 @@ export const createSourcererDataViewRoute = (
         body: buildRouteValidation(sourcererSchema),
       },
       options: {
+        authRequired: true,
         tags: ['access:securitySolution'],
       },
     },
     async (context, request, response) => {
       const siemResponse = buildSiemResponse(response);
       const siemClient = context.securitySolution?.getAppClient();
       const dataViewId = siemClient.getSourcererDataViewId();
+      const unsecuredSavedObjectClient = context.core.savedObjects.getClient({
+        excludedWrappers: ['security'],
+      });
       try {
         const [
           ,
           {
             data: { indexPatterns },
           },
         ] = await getStartServices();
-        const dataViewService = await indexPatterns.indexPatternsServiceFactory(
+        const auditLogger = security?.audit.asSystem(request);
```

Comment:
what is the importance of having the `auditLogger`?

Reply:
Behind the scene when you are using the saved object client without by passing the kibana security, we are auditing  every actions/events on the saved object type. Since we are by passing it, we still want to track what we are doing.